error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/PreconditionsRules.java

```diff
@@ -78,7 +79,7 @@ void after(int index, int size, String message) {
   static final class RequireNonNull<T> {
     @BeforeTemplate
     void before(T object) {
-      if (object == null) {
+      if (Refaster.anyOf(object == null, null == object)) {
```

Comment:
Instead of having to add the inverse check, we can add another Refaster rule that rewrites `null == object` to `object == null`. That allows us to simplify the code in many other places.